website/static/templates.json

```
@@ -2500,6 +2500,23 @@
       "msft"
     ]
   },
+  {
+    "title": "Azure Cosmos DB for NoSQL Quickstart - Rust",
+    "description": "A quick application to get started using Azure Cosmos DB for NoSQL with the Azure SDK for Rust.",
+    "preview": "./templates/images/cosmos-db-nosql-rust-quickstart.png",
+    "website": "https://github.com/azure-samples/cosmos-db-nosql-rust-quickstart",
```

```suggestion
    "website": "https://learn.microsoft.com/azure/cosmos-db/nosql/quickstart-rust",
```

Per [#471 (comment)](https://github.com/Azure/awesome-azd/pull/471#discussion_r1934519677)